VK_App/VK_App/Model/NewsFeedResponse.swift

```diff
@@ -6,15 +6,18 @@ import Foundation
 /// Новости, группы, друзья
 struct NewsFeedResponse: Decodable {
     /// Новости
-    let news: [NewsFeed]
+    var news: [NewsFeed]
     /// Группы
     let groups: [Group]
     /// Друзья
     let friends: [User]
+    /// xn
```

Comment:
```suggestion
    /// 
```
Добавь документацию нормальную